src/Views/NotFound.php

```diff
@@ -79,7 +79,7 @@ private function findNewUri($uri)
             }
         }
 
-        $baseDir = $this->container->get('settings')['directory'];
+        $baseDir = '/';
```

Comment:
This might not be right for the redirects if the were to be put in a subdirectory.. that would cause it to redirect to the root of the domain?